.github/workflows/vagrant_up.yaml

```diff
@@ -13,14 +13,16 @@ jobs:
     runs-on: macos-latest
 
     steps:
-      - uses: actions/checkout@v4
-      # Installing this is skipped in CI as using it was causing some corruption in
-      # attempting to mount the shared folder. The guest and host guest additions
-      # versions are close enough that we are fine without it.
-      # - run: vagrant plugin install vagrant-vbguest
-      - run: NO_SUBMISSIONS=1 vagrant up ubuntu-22.04
-      - name: Validate image
-        run: curl --show-error --fail --include http://localhost:1511
+      - uses: actions/checkout@v3
```

Comment:
It looks like you might have had a bad merge conflict here.  This should be bumped to `v4` to match the latest version on main.